api/graphql/schema.py

```diff
@@ -15,7 +17,14 @@ def Schema(models):
                     args={},
                     resolve=get_test_field,
                     description="Test field",
-                )
+                ),
+                "challenges": challenges_field(models),
```

Comment:
You might be asking "why call this `challenges` and not `workweek_hustles`? I think the idea is to eventually expose lots of different challenges in a single listing for clients to retrieve all at once, so I'm keeping this name generic; but that's not a strong opinion here or anything, happy to change it back if you think that's better.